store/store.go

```diff
@@ -1128,7 +1173,12 @@ type PaymentInformation struct {
 
 // PaymentMethods gets a list of the individual payment methods the user has registerd against their Ubuntu One account
 func (s *Store) PaymentMethods(user *auth.UserState) (*PaymentInformation, error) {
-	resp, err := s.doRequest(s.client, "GET", s.paymentMethodsURI.String(), halJsonContentType, "", nil, user)
+	reqOptions := &requestOptions{
+		Method: "GET",
+		Url:    s.paymentMethodsURI.String(),
```

Comment:
Looks like much more often than not this is a url.URL being transformed into a string. Should the field itself be *url.URL?